aws-sdk-core/lib/aws-sdk-core/cloudfront/url_signer.rb

```diff
@@ -18,14 +18,7 @@ module CloudFront
     #     )
     #
     class UrlSigner
-
-      # @option options [String] :key_pair_id
-      # @option options [String] :private_key
-      # @option options [String] :private_key_path
-      def initialize(options = {})
-        @key_pair_id = key_pair_id(options)
-        @private_key = private_key(options)
-      end
+      include Signer
 
       # create a signed Amazon CloudFront URL
```

Comment:
Since `Signer` module is available, then this `#signed_url` might doesn't need to duplicate the code?